.github/workflows/autopublish.yml

```
@@ -0,0 +1,20 @@
+name: CI
+on:
+  pull_request: {}
+  push:
+    branches: [main]
+
+jobs:
+  main:
+    name: Build, Validate, and Publish
+    runs-on: ubuntu-20.04
+    steps:
+      - uses: actions/checkout@v2
+      - uses: w3c/spec-prod@v2
+        with:
+          TOOLCHAIN: bikeshed
+          GH_PAGES_BRANCH: gh-pages
+          W3C_ECHIDNA_TOKEN: ${{ secrets.ECHIDNA_TOKEN }}
+          W3C_WG_DECISION_URL: https://lists.w3.org/Archives/Public/public-webappsec/2015Mar/0170.html
+          W3C_BUILD_OVERRIDE: |
+            shortname: secure-contexts
```

Oops, I think we want "CRD", right? 

```suggestion
            shortname: secure-contexts
            specStatus: CRD
```

---

> Oops, I think we want "CRD", right?

Yeah